### ISSUE TITLE
bug: unclosed file handle in VoxCPM.from_local() when reading config.json

### ISSUE THREAD

**Issue by kuishou68** [OpenBMB/VoxCPM#235]
## Summary

`VoxCPM.from_local()` in `src/voxcpm/model/voxcpm.py` opens `config.json` without a context manager, leaving the file handle unclosed:

## Location

`src/voxcpm/model/voxcpm.py`, line 851:

```python
# CURRENT — file handle never closed
config = VoxCPMConfig.model_validate_json(open(os.path.join(path, "config.json")).read())
```

## Impact

- The file descriptor is never explicitly closed; it relies on garbage collection.
- In CPython this is usually fine, but it is flagged by `ResourceWarning` and static analysers (pyflakes, flake8, pylint, ruff `SIM115`).
- In long-running servers, large model loading loops, or when many models are loaded consecutively, unclosed handles can exhaust the OS file descriptor limit.
- The missing `encoding="utf-8"` argument also makes the code locale-dependent, which can cause decode errors on Windows or non-UTF-8 locales.

## Fix

Replace with a proper context manager:

```python
with open(os.path.join(path, "config.json"), "r", encoding="utf-8") as _cfg_f:
    config = VoxCPMConfig.model_validate_json(_cfg_f.read())
```

A PR with this fix will be submitted.
